testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jaxrs/cfg/ResteasyAttributeTestCase.java

```diff
@@ -207,11 +226,32 @@ static void resetAttributeValues() throws IOException {
         setAttributeValue(JaxrsAttribute.RESTEASY_WIDER_REQUEST_MATCHING, JaxrsAttribute.RESTEASY_WIDER_REQUEST_MATCHING.getDefaultValue());
     }
 
-    static void setAttributeValue(SimpleAttributeDefinition attribute, ModelNode value) throws IOException {
+    static void setAttributeValue(AttributeDefinition attribute, ModelNode value) throws IOException {
+        ModelNode expectedValue = null;
+        switch (attribute.getType()) {
+
+            case LIST:
+                expectedValue = textifyList(value);
+                break;
+
+            case OBJECT:
+                expectedValue = textifyMap(value);
+                break;
+
+            default:
+                expectedValue = value;
```

Comment:
I believe here that you are checking what RestEasy pulls out of itself rather than the model values. The former will have resolved expressions, while the latter will have the full expression.

So in order to not compare apples and oranges, I think here you need to do something like
```
expectedValue = value.resolve();
```
This does more or less the same as the AttributeDefinition.resolve() stuff earlier, but does not have the extra stuff. Still as this is test code it should be enough.

You will need to do the same in your textifyList/Map() methods

Reply:
Thanks, @kabir. I didn't know I had to deal explicitly with expressions. I'll look into that.

Reply:
Done